packages/rich-text/src/plugins/Hyperlink/ScheduleTooltipContent.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { css, cx } from 'emotion';
+import { ScheduledAction } from '@contentful/field-editor-reference/dist/types';
+import tokens from '@contentful/forma-36-tokens';
+import { Tag, Paragraph } from '@contentful/forma-36-react-components';
+import { formatDateAndTime } from '../../helpers/formatDateAndTime';
+
+const styles = {
+  // @ts-expect-error
+  statusTag: css({
+    marginLeft: tokens.spacingM,
+    zIndex: '0 !important',
+  }),
+  positiveColor: css({
+    color: tokens.colorPositive,
+  }),
+  secondaryColor: css({
+    color: tokens.colorElementDarkest,
+  }),
+  marginRightXS: css({
+    marginRight: tokens.spacing2Xs,
+  }),
+  paragraph: css({
+    textAlign: 'center',
+    color: tokens.colorElementDarkest,
+  }),
+  time: css({
+    color: tokens.colorElementLight,
+  }),
+};
+
+export function ScheduleTooltipContent({
+  job,
+  jobsCount,
+}: {
+  job: ScheduledAction;
+  jobsCount: number;
+}) {
+  let colorPalette = '';
```

Comment:
nit: Would look better as a separate function like `const colorPalette = getColor(job.action.toLowerCase())`

Reply:
I can improve it on the next PR then, this PR is green 👀